src/components/utils/ConfirmationWindow/ConfirmationWindow.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <PopupDialog
+    v-if="isOpened"
+    @close="close()"
+  >
+    <div class="confirmation-window__wrapper">
+      <div class="confirmation-window__title">
+        {{ title }}
+      </div>
+      <div class="confirmation-window__description">
+        {{ description }}
+      </div>
+      <div class="confirmation-window__buttons-wrapper">
+        <UiButton
+          :submit="submit"
+          :warning="deletion"
+          :content="continueButtonText"
+          class="confirmation-window__continue-button"
+          @click="() => {
+            onConfirm();
+            close();
+          }"
+        />
+        <UiButton
+          secondary
+          content="Cancel"
+          @click="close()"
+        />
+      </div>
+    </div>
+  </PopupDialog>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import PopupDialog from '../PopupDialog.vue';
+import UiButton from '../UiButton.vue';
+import ConfirmationWindowOptions from './ConfirmationWindowsOptions';
+
+export default Vue.extend({
```

Comment:
Please add an example of usage. Maybe README with docs in the folder of this component